app_dart/lib/src/model/appengine/task.dart

```diff
@@ -42,6 +44,7 @@ class Task extends Model<int> {
     this.buildNumberList,
     this.builderName,
     this.luciBucket,
+    this.serializedProperties,
```

Comment:
This doesn't appear to be set anywhere. Additionally, you'll likely want this to take in `properties` and create the `serializedProperties` in the constructor from it

Reply:
Talked offline. `Task.fromTarget` is correctly parsing the properties.

We should still remove `serializedProperties` from here as it's easy to mess up (or to pass in Map<String, dynamic> properties and encode it in the constructor)

Reply:
More offline discussion. Here's a quick example

```dart
class Task {
  Task(Map<String, dynamic> properties) {
    serializedProperties = jsonEncode(properties);
  }
  
  String? serializedProperties;
}
```